experimentA/idr0062-experimentA-bulkmap-config.yml

```diff
@@ -0,0 +1,92 @@
+---
+name: idr0062-blin-nuclearsegmentation/experimentA
+version: 1
+
+defaults:
+  # Should the column be processed when creating bulk-annotations (yes/no)
+  include: no
+  # Columns type of the bulk-annotations column
+  type: string
+
+  # If non-empty a string used to separate multiple fields in a column
+  # White space will be stripped
+  split: ;
+  # Should this column be included in the clients (yes/no)
+  includeclient: yes
+  # Should this column be visible in the clients, if no the column should be
+  # hidden in the client but will still be indexed by the searcher (yes/no)
+  visible: yes
+  # Should empty values be omitted from the client display
+  omitempty: yes
+
+columns:
+
+  - name: Protocol 1
+    clientname: Protocol 1
+    include: yes
+  - name: Protocol 2
+    clientname: Protocol 2
+    include: yes
+  - name: Protocol 3
+    clientname: Protocol 3
+    include: yes
+  - name: Experimental Condition [Genotype]
+    clientname: Genotype
+    include: yes
+  - name: Comment [Experiment]
+    clientname: Experiment Comment
+    include: yes
+  - name: Comment [Image File Type]
+    clientname: Image File Type Comment
+    include: yes
+  - name: Segmented Image
+    clientname: Segmented Image
+    include: yes
+  - name: Segmented Channel
+    clientname: Segmented Channel
+    include: yes
+  - name: Segmentation Method
+    clientname: Segmentation Method
+    include: yes
+
+  ######################################################################
+  # mapr groups
+  ######################################################################
+
+  - group:
+      namespace: openmicroscopy.org/mapr/organism
+      columns:
+        - name: Characteristics [Organism]
+          clientname: Organism
+          include: yes
+
+  - group:
+      namespace: openmicroscopy.org/mapr/gene
+      columns:
+        - name: Comment [Gene Identifier]
+          clientname: Gene Identifier
+          include: yes
```

Comment:
You might want to add the following block as well

```
      - name: Comment [Gene Identifier]
        clientname: Gene Identifier URL
        clientvalue: http://www.ensembl.org/id/{{ value|urlencode }}
        include: yes
```

to create an icon hyperlinking to ensembl.org